montrek-software/montrek
# 506 generate excels for download on parallel worker task

# How To test:

- Start up docker and inspect worker logs:

```
docker compose logs parallel_worker
```

Every Table manager should have a task instance registered:

 
![image](https://github.com/user-attachments/assets/dc49aa5a-74da-41df-ae37-93b6a1d44384)


Go To montrek, pick up a few large tables and trigger the Excel or csv downloads:

 
![image](https://github.com/user-attachments/assets/053e7f5a-c372-4c24-afbc-33c8ead262d1)

After a while they should be finished:

![image](https://github.com/user-attachments/assets/5356d88e-d2e6-4cd2-b2d6-5211b37bbb11)

You should have received some mails:

![image](https://github.com/user-attachments/assets/72674509-fef2-435a-ae04-283d50cfbb06)
